# The decriptions of all received robots
RobotDescription[] descriptions